docs/porting-guides/6.x.x.md

```diff
@@ -156,6 +156,25 @@ snmp_settings:
       ipv4_acl: IPV4
 ```
 
+### Configure BGP AS notation to Asdot/Asplain using `bgp_as_notation`
+
+In AVD 5.x, All BGP AS would get converted to `asplain` format in EOS even when it is given in `asdot` format in eos_designs input, since the option to set AS notation was not available in eos_designs.
+
+Starting AVD 6.0.0, BGP AS notation can be set by defining `bgp_as_notation` to `auto`, `asdot` or `asplain` default value is `auto`.
+
+If `bgp_as_notation` is set to `auto` or left undefined, the system examines its own AS number.
+
+- If the AS number contains a dot (`.`), all other AS numbers get converted to `asdot` notation.
+- If the AS number does not contain a dot, all other AS numbers get converted to `asplain` notation.
+
+```diff
++ bgp_as_notation: asdot
```

Comment:
the porting guide should indicate how to retain the old behavior so probably should say - "to retain old behavior, use `asplain`"

Reply:
added